fdbserver/Knobs.cpp

```diff
@@ -363,7 +363,8 @@ void ServerKnobs::initialize(bool randomize, ClientKnobs* clientKnobs, bool isSi
 	init( START_TRANSACTION_MAX_REQUESTS_TO_START,             10000 );
 	init( START_TRANSACTION_RATE_WINDOW,                         2.0 );
 	init( START_TRANSACTION_MAX_EMPTY_QUEUE_BUDGET,             10.0 );
-	init( START_TRANSACTION_MAX_QUEUE_SIZE,                      1e6 );
+	init( START_SYSTEM_AND_DEFAULT_TRANSACTION_MAX_QUEUE_SIZE,   1e6 );
+	init( START_BATCH_TRANSACTION_MAX_QUEUE_SIZE,                1e5 ); if( randomize && BUGGIFY ) START_BATCH_TRANSACTION_MAX_QUEUE_SIZE = 1000;
```

Comment:
Just curious, did you weigh this idea against one where you maintain a single limit for the combined queues? The main benefit of that would be that you could have a larger capacity for batch transactions, while the drawback would be that you'd have to pop items off the batch queue to make room for default transactions when the queues are full.

Given that batch transactions are more likely to fill up the queue, a 10x decrease could represent a behavior change for someone, especially if we are porting this back to old releases. That said, it may not be that big a deal.